### ISSUE TITLE
Connection to SQL 2005 SP4

### ISSUE THREAD

**Issue by lestevam** [microsoft/azuredatastudio#765]
- SQL Operations Studio Version: 0.26.7

Is it possible to connect to a SQL Server 2005 with SP4? Has anyone ever been able to connect?

**Comment by kburtram** [comment]
@lestevam we don't include SQL Server 2005 SP4 in our test matrix.  Are you getting any particular error while trying to connect?

**Comment by lestevam** [comment]
@kburtram I made some tests to try to give more useful information, I use the linux antergos (base arch) and the first version of the client does not work with SQL 2005SP4. Searching only found information about the obligation to have SP4 installed in the 2005 version.

When trying to reconnect now with the client in version 0.26.7 on this my machine the connection was successful, so I went to try on other machines. In Mac IOs the connection was also established successfully, but when testing on two ubuntu machines with unit, they can not connect to SQL2005SP4, as if they were missing some dependency.

Both make the same error as the image.

![errorsqlopsubuntu](https://user-images.githubusercontent.com/917757/36917075-8e2d8cc6-1e34-11e8-864b-3a7de5697e01.png)

The strangest thing is that if I try a connection by 'telnet' to the same server and in the standard SQL port the connection is established.
